src/providers/mod.rs

```diff
@@ -52,6 +52,10 @@ use crate::network;
 const ENV_PREFIX: &str = "AFTERBURN_";
 #[cfg(feature = "cl-legacy")]
 const ENV_PREFIX: &str = "COREOS_";
+// AFTERBURN_SSH_AUTHORIZED_KEYS_MESSAGEID keeps track of
```

Comment:
Docstrings are marked with `///`; to need to repeat the constant name:
`/// Message ID marker for authorized-keys entries in journal.`